src/NuGet.Core/NuGet.Packaging/PackageReaderBase.cs

```diff
@@ -527,7 +526,9 @@ protected static string GetNuspecFile(IEnumerable<string> files)
 
             if (nuspecPaths.Count == 0)
             {
-                throw new PackagingException(Strings.MissingNuspec);
+                throw new PackagingException(NuGetLogCode.NU5037, string.Format(
```

Comment:
`Strings.Error_MissingNuspecFile` doesn't have any `{0}`, so the `string.Format` is unnecessary.